Experiments/QKS/2019_03_05_qiskit_implementation/main.py

```diff
@@ -23,6 +24,16 @@ def logistic_regression(X_train, X_test, y_train, y_test):
 
     return train_preds, test_preds
 
+
+def run_qks(p, number_of_qubits, qc, n_episodes, scale, n_trials):
```

Comment:
I have no idea what `p` and `r` mean - please use some more descriptive variable names.

Reply:
Now I remember - `p` and `r` were some parameters of QKS, notation used in the paper.
Please add some documentation to this function, describing what's what + indicating that this reflects the notation from the paper.